roles/dbserver/tasks/main.yml

```diff
@@ -10,11 +10,3 @@
     mode: 0600
   become: yes
   become_user: "{{ unicorn_user }}"
-
-# Rails expects unicode databases. See the script for more details.
-- name: fix postgres db encoding problem
```

Comment:
This means we can't use `rake db:reset` on staging servers?

I ran into this problem only once. It usually doesn't happen with English data but the application can crash when using other locales and the database is not UTF8.

I'm wondering if using template0 in database.yml could help: https://stackoverflow.com/a/16737073/3377535

Reply:
Ah, I forgot about `db:reset` on staging :see_no_evil: . Ok, I the template hack will have to go back in for sure, but maybe we can do it with Ansible in a clearer way.